packages/nestjs-auth-github/src/interfaces/auth-github-settings.interface.ts

```diff
@@ -1,9 +1,14 @@
 import { AuthenticationCodeInterface } from '@concepta/ts-common';
 import { Type } from '@nestjs/common';
+import { AuthGithubProfileInterface } from './auth-github-profile.interface';
+import { AuthGithubSignInterface } from './auth-github-sign.interface';
 
 export interface AuthGithubSettingsInterface {
   clientId: string;
   clientSecret: string;
   callbackURL: string;
   loginDto?: Type<AuthenticationCodeInterface>;
+  profileFormatter: (
```

Comment:
this needs to be a type defined in a separate file and exported from the project